pkg/api/mlflow/service/artifact/storage/gs.go

```diff
@@ -25,10 +26,16 @@ type GS struct {
 }
 
 // NewGS creates new GC instance.
```

Comment:
Abbreviations too brief?

Reply:
yep, changed comments to use `Google Storage` instead of `GS`.